web/profiles/joinup/config/install/search_api.server.collections_search_server.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 9fe2d8ff-5921-4ec5-a400-c5351a1499f4
+langcode: en
+status: true
+dependencies:
+  module:
+    - search_api_db
+id: collections_search_server
+name: 'Collections Search Server'
```

Comment:
This seems to imply we are dedicating an entire server to just indexing collections. I need to refresh my memory on how Search API works, but I think this server can be used for indexing the entire project, so we can call it the "Joinup Search Server".